python/src/addons/ReadOnlyIndex.pyx

```diff
@@ -0,0 +1,36 @@
+
+
+    def get (self, key, default = None):
```

Comment:
I see that we use lowercase `get()` to have `dict` like interface, but I've noticed for `Dictionary()` users often times it's quite confusion that all methods are uppercase except one. Maybe it's worth to have `Get()` here ?